.devcontainer/chefs_local/docker-compose.yml

```diff
@@ -39,4 +38,4 @@ services:
       - "-Dkeycloak.migration.action=import"
       - "-Dkeycloak.migration.provider=singleFile"
       - "-Dkeycloak.migration.file=/tmp/realm-export.json"
-      - "-Dkeycloak.migration.strategy=OVERWRITE_EXISTING"
\ No newline at end of file
+      - "-Dkeycloak.migration.strategy=OVERWRITE_EXISTING"
```

Comment:
Maybe this is a difference in how Prettier is applied. I'll see if I can standardize it so that we don't have this kind of churn in YAML files.